cpp/CMakeLists.txt

```diff
@@ -151,10 +138,6 @@ include(${rapids-cmake-dir}/cpm/cuco.cmake)
 rapids_cpm_cuco(BUILD_EXPORT_SET cugraph-exports INSTALL_EXPORT_SET cugraph-exports)
 include(cmake/thirdparty/get_raft.cmake)
 
-if(USE_CUGRAPH_OPS)
-  include(cmake/thirdparty/get_libcugraphops.cmake)
```

Comment:
This whole file should be removed too now, this was the only use of it: https://github.com/rapidsai/cugraph/blob/branch-24.12/cpp/cmake/thirdparty/get_libcugraphops.cmake

Reply:
Removed